src/grb_env.jl

```diff
@@ -3,14 +3,22 @@
 
 type Env
     ptr_env::Ptr{Void}
-    
-    function Env()
+
+    function Env(cloudhost::ASCIIString="", cloudpassword::ASCIIString="")
         a = Array(Ptr{Void}, 1)
-        ret = @grb_ccall(loadenv, Cint, (Ptr{Ptr{Void}}, Ptr{UInt8}), 
-            a, C_NULL)
+        if length(cloudhost) > 0 # cloud server
+            ret = @grb_ccall(loadclientenv, Cint,
+                (Ptr{Ptr{Void}}, Ptr{UInt8}, Ptr{UInt8}, Cint, Ptr{UInt8}, Cint, Cdouble),
+                a, C_NULL, cloudhost, -1, cloudpassword, 0, -1)
+        else # local installation
+            ret = @grb_ccall(loadenv, Cint, (Ptr{Ptr{Void}}, Ptr{UInt8}),
+                a, C_NULL)
+        end
         if ret != 0
             if ret == 10009
                 error("Invalid Gurobi license")
+            elseif ret == 10022
+                error("Problem communicating with the Gurobi Compute Server")
```

Comment:
Could document the LD_LIBRARY thing here - no harm in having verbose error messages.